<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1639702274221' stop='1639702274305'>
    <name>My Login application succes</name>
    <title>My Login application succes</title>
    <test-cases>
        <test-case start='1639702274221' status='broken' stop='1639702274254'>
            <name>should login with valid credentials</name>
            <title>should login with valid credentials</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-1'/>
                <label name='feature' value='Login'/>
                <label name='severity' value='Critical'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-96.0.4664.110'/>
            </parameters>
            <steps>
                <step start='1639702274224' status='broken' stop='1639702274254'>
                    <name>POST /session/:sessionId/url</name>
                    <title>POST /session/:sessionId/url</title>
                    <attachments>
                        <attachment title='Request' source='a7b16123-a4e7-4d8c-90bc-c3bf286630da-attachment.json' type='application/json' size='45'/>
                    </attachments>
                    <steps>
                        <step start='1639702274239' status='broken' stop='1639702274254'>
                            <name>GET /session/:sessionId/screenshot</name>
                            <title>GET /session/:sessionId/screenshot</title>
                            <attachments/>
                            <steps/>
                        </step>
                    </steps>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>connect ECONNREFUSED 127.0.0.1:9515</message>
                <stack-trace>RequestError: connect ECONNREFUSED 127.0.0.1:9515
                    at ClientRequest.&lt;anonymous> (C:\Users\rcruz\OneDrive\Escritorio\PruebaFinal_CursoWdio\node_modules\got\dist\source\core\index.js:962:111)
                    at Object.onceWrapper (events.js:421:26)
                    at ClientRequest.emit (events.js:326:22)
                    at ClientRequest.EventEmitter.emit (domain.js:483:12)
                    at ClientRequest.origin.emit (C:\Users\rcruz\OneDrive\Escritorio\PruebaFinal_CursoWdio\node_modules\@szmarczak\http-timer\dist\source\index.js:43:20)
                    at Socket.socketErrorListener (_http_client.js:427:9)
                    at Socket.emit (events.js:314:20)
                    at Socket.EventEmitter.emit (domain.js:483:12)
                    at emitErrorNT (internal/streams/destroy.js:92:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
                    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1144:16)
                    at TCPConnectWrap.callbackTrampoline (internal/async_hooks.js:126:14)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1639702274256' status='broken' stop='1639702274305'>
            <name>should logout and verify logout</name>
            <title>should logout and verify logout</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-1'/>
                <label name='feature' value='Login'/>
                <label name='severity' value='Critical'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-96.0.4664.110'/>
            </parameters>
            <steps>
                <step start='1639702274261' status='broken' stop='1639702274304'>
                    <name>POST /session/:sessionId/url</name>
                    <title>POST /session/:sessionId/url</title>
                    <attachments>
                        <attachment title='Request' source='b6598069-1cef-4c2a-a7d1-5032673d6037-attachment.json' type='application/json' size='45'/>
                    </attachments>
                    <steps>
                        <step start='1639702274289' status='broken' stop='1639702274304'>
                            <name>GET /session/:sessionId/screenshot</name>
                            <title>GET /session/:sessionId/screenshot</title>
                            <attachments/>
                            <steps/>
                        </step>
                    </steps>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>connect ECONNREFUSED 127.0.0.1:9515</message>
                <stack-trace>RequestError: connect ECONNREFUSED 127.0.0.1:9515
                    at ClientRequest.&lt;anonymous> (C:\Users\rcruz\OneDrive\Escritorio\PruebaFinal_CursoWdio\node_modules\got\dist\source\core\index.js:962:111)
                    at Object.onceWrapper (events.js:421:26)
                    at ClientRequest.emit (events.js:326:22)
                    at ClientRequest.EventEmitter.emit (domain.js:483:12)
                    at ClientRequest.origin.emit (C:\Users\rcruz\OneDrive\Escritorio\PruebaFinal_CursoWdio\node_modules\@szmarczak\http-timer\dist\source\index.js:43:20)
                    at Socket.socketErrorListener (_http_client.js:427:9)
                    at Socket.emit (events.js:314:20)
                    at Socket.EventEmitter.emit (domain.js:483:12)
                    at emitErrorNT (internal/streams/destroy.js:92:8)
                    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
                    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1144:16)
                    at TCPConnectWrap.callbackTrampoline (internal/async_hooks.js:126:14)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>